README.md

```diff
@@ -25,32 +27,36 @@ cd opg-lpa
 
 ### Install pre-commit hooks (mac)
 
-Install the precommit hooks as follows in the root of the repo directory (Mac only):
+Install the precommit hooks and dependencies in the root of the repo directory:
 
 ```bash
 brew install golang
-brew install php-code-sniffer
+
+# python code linting
+brew install black
+
+# javascript code linting
+brew install eslint
+
+# PHP code linting
 brew install php-cs-fixer
-brew install phpstan
+
 brew install pre-commit
+
 pre-commit install
 ```
 
-This will install the pre-commit hooks for the repo. This covers:
+Pre-commit hooks run any time you add a commit. They covers:
 
-* PHP code fixers
+* PHP code formatting and fixing
+* Python code linting
+* JavaScript code linting
 * Terraform
 * Secrets commit detection (AWS, general secrets)
 * Whitespace and end of file fixers
 
 Add more as needed to the `.pre-commit-config.yaml`.
 
-For Linux users, revert to the instructions for installing phpcs, phpstan and pre-commit hooks:
-
-* <https://github.com/squizlabs/PHP_CodeSniffer>
-* <https://phpstan.org/user-guide/getting-started>
-* <https://pre-commit.com/index.html#install>
-
```

Comment:
Is this just no longer relevant, or are we forgetting about running on linux for now?

Reply:
I'm taking out the Linux instructions. Anyone who is Linux savvy would be able to convert these instructions to do the installation.

Reply:
👍